examples/extending-objects/Polymorphism/Go/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import "fmt"
+
+func main() {
```

Comment:
Ensure consistency in documentation across all examples (see what you did here for a comparison: https://github.com/johnfraserss/ICS4U/pull/36/files#diff-ecef7c58901f46a0422a271ea876d9db220e8203902c51fce2ea1226b04624ea )